src/corelibs/U2Core/src/gobjects/MultipleAlignmentObject.h

```diff
@@ -173,6 +173,10 @@ class U2CORE_EXPORT MultipleAlignmentObject : public GObject {
 
     void saveState();
     void releaseState();
+    /*
+    * Returns true if there is removable gaps in alignment.
```

Comment:
nit: The comment does not correlate with the method name and is focused on a specific scenario.